plugins/slo/pkg/slo/metrics.go

```diff
@@ -39,21 +41,27 @@ type goodQueryPlaceholder struct {
 	Filter   string
 }
 
+type metricTemplate struct {
+	NameRegex string
+	ServiceId string
+}
+
 func InitMetricList() {
```

Comment:
Would this be better as an `init()` method?

Reply:
I think putting this as part of the `plugin` startup code would be best. Is there an existing place to do that

Reply:
Oh I see `init` has a special meaning in go, yeah let me put the contents of InitMetricList into `init`, but also I'll put the metric list in the (k,v) store when we first set the storage in the SLO plugin

Reply:
Did a minor refactor, put `InitMetricList` into the special `init` function in this module, and removed references to it in the api package, preferring to set it once (and not check in each method if it is initialized) when we set the storage future for the SLO plugin